postman/src/main/java/com/workday/postman/codegen/BoxableSaveStatementWriter.java

```diff
@@ -52,6 +50,8 @@ public void writeFieldWriteStatement(VariableElement field, JavaWriter writer)
     }
 
     private String getSaveType(TypeMirror type) {
-        return StringUtils.capitalize(metaTypes.asPrimitive(type).toString());
+        String typeString = metaTypes.asPrimitive(type).toString();
+        if (typeString == null || typeString.length() == 0) return typeString;
```

Comment:
`toString()` never returns null, and in this case, will never return an empty string, so you can remove the `if` block.